medicines/api/src/document.rs

```diff
@@ -55,31 +39,235 @@ pagination! {Documents, DocumentEdge, Document}
 
 pub async fn get_documents(
     client: &impl Search,
-    search: Option<String>,
+    search: String,
     first: Option<i32>,
```

Comment:
Would this be better off as a `u32` as you don't want to ever pass in a negative number?